gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageUtil.java

```diff
@@ -2,15 +2,11 @@
 
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
-import static com.google.common.base.Strings.isNullOrEmpty;
-
-import com.google.appengine.tools.cloudstorage.GcsFileOptions;
 
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.file.Path;
 import java.nio.file.ProviderMismatchException;
-import java.util.Map;
 import java.util.regex.Pattern;
 
 final class CloudStorageUtil {
```

Comment:
remove internal bug reference. If desired create a similar github issue and replace the reference with it.